src/graphql/types/abstract/wallet.ts

```diff
@@ -24,7 +28,7 @@ const IWallet = GT.Interface({
       description: dedent`Transactions are ordered anti-chronologically,
       ie: the newest transaction will be first`,
       type: TransactionConnection,
-      args: connectionArgs,
+      args: { ...connectionArgs, hash: { type: PaymentHash } },
```

Comment:
How can I add multiple types here? This should be a union of `PaymentHash | OnChainTxHash` (cc @samerbuna, @dolcalmi)

Reply:
Yea I think I'll actually pause this one because #1210 may be a cleaner way to get at this information